r_markdown/emissions.Rmd

```diff
@@ -442,13 +548,6 @@ ___
   ions <- dplyr::mutate(ions, 
                         mass_emitted = (conc - conc_bg) * flow_hood * dur,
                         mass_carbon = mass_emitted * 0)
-
-  ions_g <- dplyr::group_by(ions, id) %>%
```

Comment:
Stop it stop it stop it!!!!!! This is either a great Nick wide-up or something I don't want to consider!!!